pkgs/by-name/pa/packetry/package.nix

```
@@ -0,0 +1,63 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  rustPlatform,
+  gtk4,
+  pkg-config,
+  pango,
+  wrapGAppsHook4,
+  darwin,
+  versionCheckHook,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "packetry";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "greatscottgadgets";
+    repo = "packetry";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-eDVom0kAL1QwO8BtrJS76VTvxtKs7CP6Ob5BWlE6wOM=";
+  };
+
+  cargoHash = "sha256-xz9PdVVB1u6s/anPBRonWS1kMN+4kfkK/gaOlF9Z3yk=";
+
+  nativeBuildInputs = [
+    pkg-config
+    wrapGAppsHook4
+  ];
+
+  buildInputs =
+    [
+      gtk4
+      pango
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      darwin.apple_sdk_11_0.frameworks.AppKit
+    ];
```

```suggestion
    ];
```
We don't need to declare specific frameworks anymore. 

Does it actually need apple-sdk_11 or can it build with lower SDK?

---

> We don't need to declare specific frameworks anymore.

Thanks, I updated this.

> Does it actually need apple-sdk_11 or can it build with lower SDK?

It actually needs 10.14+ (due to [dark-light](https://github.com/greatscottgadgets/packetry/blob/v0.4.0/Cargo.toml#L55)'s dependency on [NSAppearanceName*DarkAqua](https://developer.apple.com/documentation/appkit/nsappearancenameaccessibilityhighcontrastdarkaqua?language=objc)), but apple-sdk_11_14 is not available for aarch64-darwin, so I wasn't sure if I should add a condition here like [it's done in qt](https://github.com/carlossless/nixpkgs/blob/d051f421fb8f21dc591656a417a508afbf92bd5a/pkgs/development/libraries/qt-5/modules/qtbase.nix#L44) and left it like this.

---

Thanks for looking it it, this should be fine.